### ISSUE TITLE
Godot freezes when create a new animation on node AnimatedSprite2D

### ISSUE THREAD

**Issue by ryugold777** [godotengine/godot#79677]
### Godot version

4.1

### System information

Arch Linux\Linux

### Issue description

Every time that I create an animation on node AnimatedSprite2D and try to rename godot freezes and I need to kill the process on terminal.

### Steps to reproduce

create a new animation and rename.

### Minimal reproduction project

none

**Comment by Calinou** [comment]
Can you reproduce this in 4.1.1? See https://github.com/godotengine/godot/issues/79012.

**Comment by ryugold777** [comment]
On xorg Works! But is this a bug?

**Comment by Calinou** [comment]
> On xorg Works! But is this a bug?

Yes, it's an XWayland issue which we probably can't fix: https://github.com/godotengine/godot/issues/78581
